### PR TITLE
Attempt to upload data to database, data currently shows up in post request body, but still struggling to get it to insert the data into the database

### PR DESCRIPTION
**Pull request by bethanneb**

### CODE REVIEW

File: db-migrations/3_test3.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS data_upload (
```
Comment (gik0geck0) [inline db-migrations/3_test3.sql:1]: is this just a temp table while you figure it out?

Comment (bethanneb) [reply db-migrations/3_test3.sql:1]: Yes

File: src/server/api.ts [hunk 2]
```diff
@@ -6,10 +6,16 @@ import path from 'path';
 
 //to import queries from DB service
 import { testQueryAll, testQueryAll2} from './db_setup';
```
Comment (gik0geck0) [inline src/server/api.ts:8]: We should pick better names for these methods, like `getGameResultsFromDb`

**Comment by bethanneb** [comment]
Closed this pull request so I could open a new one

**Pull request closed without merging**